<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EQ-EQ6283668" ignored="0" name="testngTest" tests="4" failures="1" timestamp="14 Oct 2018 13:14:58 GMT" time="0.074" errors="0">
  <testcase name="firstTestMethod" time="0.018" classname="TstNg.TestRetryXmlTestCases"/>
  <testcase name="secondTestMethod" time="0.004" classname="TstNg.TestRetryXmlTestCases">
    <skipped/>
  </testcase> <!-- secondTestMethod -->
  <testcase name="secondTestMethod" time="0.003" classname="TstNg.TestRetryXmlTestCases">
    <skipped/>
  </testcase> <!-- secondTestMethod -->
  <testcase name="secondTestMethod" time="0.003" classname="TstNg.TestRetryXmlTestCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TstNg.TestRetryXmlTestCases.secondTestMethod(TestRetryXmlTestCases.java:16)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- secondTestMethod -->
</testsuite> <!-- testngTest -->
